source/api/content-state/0.4/index.md

```diff
@@ -0,0 +1,701 @@
+  "motivation": ["contentState"],
+  "target": {
+    "type": "SpecificResource",
+    "source": {
+        "id": "https://example.org/iiif/id1/canvas1",
+        "type": "Canvas",
+        "partOf": [{
+            "id": "https://example.org/iiif/id1/manifest",
+            "type": "Manifest"
+        }]
+    },
+    "selector": {
+        "type": "PointSelector",
+        "t": 14.5
+	  }
+  }
+}
+```
+
+This example should cause a viewer to open Manifest https://example.org/iiif/id1/manifest, navigate to Canvas https://example.org/iiif/id1/canvas1, and start playing at 14.5 seconds into that canvas.
+
+
+### 4.3. Multiple targets for a comparison view
+
+```json
+{
+  "@context": "http://iiif.io/api/presentation/{{ page.major }}/context.json",
+  "id": "https://example.org/import/3",
+  "type": "Annotation",
+  "motivation": "contentState",
+  "target": [
+      {
+          "id": "https://example.org/iiif/item1/canvas37",
+          "type": "Canvas",
+          "partOf": [
+              {
+                  "id": "https://example.org/iiif/item1/manifest",
+                  "type": "Manifest"
+              }
+          ]
+      },
+      {
+          "id": "https://example.org/iiif/item2/canvas99",
+          "type": "Canvas",
+          "partOf": [
+              {
+                  "id": "https://example.org/iiif/item2/manifest",
+                  "type": "Manifest"
+              }
+          ]
+      }
+  ]
+}
+```
+
+Here the viewer should open two manifests at once (if it is capable of such a view).
+
+
+### 4.4. Search Results
+
+The following example uses the compact, query string form of the content state to demonstrate what HTML search results linking to a particular viewer might look like.
+
+Firstly, in non-valid, unencoded form to show the annotation:
+
+```html
+<h2>Results for "cats"</h2>
+<ol>
+  <li>
+    <h3><a href='viewer.html?iiif-content={"id":"https://example.org/alice/canvas77#xywh=1000,2000,1000,2000","type":"Canvas","partOf":[{"id":"https://example.org/alice/manifest","type":"Manifest"}]}'>Alice in Wonderland</a></h3>
+    <p>...she has often seen a <b>cat</b> without a grin but never a grin without a <b>cat</b></p>
+  </li>
+  <!-- ... more results -->
+</ol>
+```
+
+...and then in encoded form:
+
+```html
+<h2>Results for "cats"</h2>
+<ol>
+  <li>
+    <h3><a href="viewer.html?iiif-content=eyJpZCI6Imh0dHBzOi8vZXhhbXBsZS5vcmcvYWxpY2UvY2FudmFzNzcjeHl3aD0xMDAwLDIwMDAsMTAwMCwyMDAwIiwidHlwZSI6IkNhbnZhcyIsInBhcnRPZiI6W3siaWQiOiJodHRwczovL2V4YW1wbGUub3JnL2FsaWNlL21hbmlmZXN0IiwidHlwZSI6Ik1hbmlmZXN0In1dfQ==">Alice in Wonderland</a></h3>
+    <p>...she has often seen a <b>cat</b> without a grin but never a grin without a <b>cat</b></p>
+  </li>
+  <!-- ... more results -->
+</ol>
+```
+
+
+## 5. Media Type of Content State Documents
+
+The format for all content state responses is JSON, as described above. It is good practice for all resources with an HTTP(S) URI to provide their description when the URI is dereferenced.
```

Comment:
This should be clearer here or somewhere else that all responses use Presentation API context as param.